src/include/storage/data_table.h

```diff
@@ -281,6 +293,25 @@ class DataTable {
   bool CompareAndSwapVersionPtr(TupleSlot slot, const TupleAccessStrategy &accessor, UndoRecord *expected,
                                 UndoRecord *desired);
 
+  /**
+   * Write a schema message
```

Comment:
Can you add a comment showing the serialized format, similar to how [`ProjectedRows`](https://github.com/cmu-db/terrier/blob/18c62295b42cc6acd06926989318bd2d94a1e416/src/include/storage/projected_row.h#L19-L23) is shown here

Reply:
I added comments describes the high-level format, for example, the first part is a Schema Message, then some RecordBatch messages + dictionary messages. But I may not be able to state their internal structures because they are all defined by flatbuffer schema files. I provided pointers to those files.